nl.json

```diff
@@ -61,9 +60,9 @@
                 "reminders" : "herinneringen",
                 "history" : "winkel geschiedenis"
             },
-            "news" : "nieuws",
+              "news" : "nieuws",
```

Comment:
The spacing was correct before. Please don't add unnecessary spaces before the translation key.